builtin/logical/pki/backend.go

```diff
@@ -136,3 +141,48 @@ The PKI backend dynamically generates X509 server and client certificates.
 After mounting this backend, configure the CA using the "pem_bundle" endpoint within
 the "config/" path.
 `
+
+func metricsKey(req *logical.Request, extra ...string) ([]string, error) {
```

Comment:
Looks like this signature could be simplified by removing the error from the return values.

Reply:
legacy to earlier work, thanks.